importer_client/python/tools/timesketch_importer.py

```diff
@@ -228,50 +176,57 @@ def upload_file(
             'Path to the file that is to be imported.'))
 
     options = argument_parser.parse_args()
-    config_options = {}
-
-    if options.config_file:
-        if not os.path.isfile(options.config_file):
-            logger.error('Config file does not exist ({0:s})'.format(
-                options.config_file))
-            sys.exit(1)
-        with open(options.config_file, 'r') as fh:
-            config_options = yaml.safe_load(fh)
 
     if not os.path.isfile(options.path):
         logger.error('Path {0:s} is not valid, unable to continue.')
         sys.exit(1)
 
-    conf_host = options.host or config_options.get('host', '')
-    if not conf_host:
-        logger.error('Hostname for Timesketch server must be set.')
-        sys.exit(1)
+    assistant = config.ConfigAssistant()
+    assistant.load_config_file()
+    assistant.load_config_dict(vars(options))
 
-    conf_password = options.password or config_options.get('password', '')
-    conf_pwd_prompt = options.pwd_prompt or config_options.get(
-        'pwd_prompt', False)
-    if not conf_password and conf_pwd_prompt:
+    conf_password = ''
+    if options.pwd_prompt:
         conf_password = getpass.getpass('Type in the password: ')
```

Comment:
Maybe add some text about what this password is?

Reply:
done